cmd/gardener-extension-shoot-oidc-service/app/options.go

```diff
@@ -38,7 +39,13 @@ func NewOptions() *Options {
 	}
 
 	webhookSwitches := oidccmd.WebhookSwitchOptions()
-	webhookOptions := webhookcmd.NewAddToManagerOptions(ExtensionName, webhookServerOptions, webhookSwitches)
+	webhookOptions := webhookcmd.NewAddToManagerOptions(
+		ExtensionName,
+		"",
```

Comment:
Why is the managed resource name here empty? I see that this is passed to to the `gcp-extension-provider` [ref](https://github.com/gardener/gardener-extension-provider-gcp/blob/621d99a427b91cca6b65a67b7225d34f368ca6fd/cmd/gardener-extension-provider-gcp/app/app.go#L129). I guess that if empty value is passed then this will not become part of this managed resource with the controlplane webhooks. Is that right and is that the desired behavior?

Reply:
I see, thanks for the explanation